ui/Modal/NewProject.svelte

```diff
@@ -153,6 +153,7 @@
     flex-direction: column;
     justify-content: center;
     text-align: center;
+    width: -webkit-fill-available;
```

Comment:
The following has the same behavior but uses standardized CSS functionality so I’d prefer it.

```suggestion
    align-self: stretch;
```

Reply:
TIL :cowboy_hat_face: 

The same behaviour can also be achieved using `width: 100%;`, I picked `width: -webkit-fill-available;` as 1) I found it more idiomatic and 2) because I see it being used in other places already.

Which of the 3 do you prefer?

Reply:
I think `width: 100%` would be the most obvious one.

Reply:
Done!